torch/_dynamo/symbolic_convert.py

```diff
@@ -1007,11 +998,13 @@ def CALL_FUNCTION_KW(self, inst):
         self.call_function(fn, args, kwargs)
 
     def LOAD_METHOD(self, inst):
+        unimplemented("LOAD_METHOD should have been converted to LOAD_ATTR")
```

Comment:
Explanation for review: Fixed an unused code path

Reply:
This is used in 3.11.